Index: xpra/xpra/os_util.py
===================================================================
--- xpra.orig/xpra/os_util.py
+++ xpra/xpra/os_util.py
@@ -318,7 +318,7 @@ def is_RedHat():
     return is_distribution_variant(b"RedHat")
 
 
-_linux_distribution = None
+_linux_distribution = ("ubuntu","18.04","bionic")
 def get_linux_distribution():
     global _linux_distribution
     if LINUX and not _linux_distribution:
