Description: fix spelling mistakes
Author: أحمد المحمودي (Ahmed El-Mahmoudy) <aelmahmoudy@users.sourceforge.net>
Bug: https://www.xpra.org/trac/ticket/2327/
Forwarded: https://www.xpra.org/trac/attachment/ticket/2327/tupos.diff

Index: xpra/man/xpra.1
===================================================================
--- xpra.orig/man/xpra.1	2019-03-28 13:13:12.000000000 +0100
+++ xpra/man/xpra.1	2019-06-13 03:20:56.121845082 +0200
@@ -1184,7 +1184,7 @@
 This option is only relevant when using a lossy encoding.
 .TP
 \fB--shortcut-modifiers\fP=\fIMODIFIERS\fP
-Defines the default shorcut modifers required by the \fIkey-shortcuts\fP,
+Defines the default shortcut modifiers required by the \fIkey-shortcuts\fP,
 these modifiers can then be referred to as \fI#\fP.
 The default value is 'auto' which evaluates to \fIMeta+Shift\fP on most
 platforms.
Index: xpra/xpra/rectangle.pyx
===================================================================
--- xpra.orig/xpra/rectangle.pyx	2019-03-28 13:13:12.000000000 +0100
+++ xpra/xpra/rectangle.pyx	2019-06-13 03:19:17.289449379 +0200
@@ -100,7 +100,7 @@
 
     def substract(self, const int x, const int y, const int w, const int h):
         """ returns the rectangle(s) remaining when
-            one substracts the given rectangle from it, or None if nothing remains
+            one subtracts the given rectangle from it, or None if nothing remains
         """
         if w==0 or h==0 or self.width==0 or self.height==0:
             #no rectangle, no change:
